governance/api.go

```diff
@@ -157,6 +157,33 @@ func (api *PublicGovernanceAPI) ItemsAt(num *rpc.BlockNumber) (map[string]interf
 	}
 }
 
+func (api *PublicGovernanceAPI) PendingChanges() map[string]interface{} {
+	return api.governance.PendingChanges()
+}
+
+func (api *PublicGovernanceAPI) Votes() []GovernanceVote {
+	return api.governance.Votes()
+}
+
+func (api *PublicGovernanceAPI) IdxCache() []uint64 {
+	return api.governance.IdxCache()
+}
+
+func (api *PublicGovernanceAPI) IdxCacheFromDb() []uint64 {
+	return api.governance.IdxCacheFromDb()
+}
+
+func (api *PublicGovernanceAPI) ItemCacheFromDb(num *rpc.BlockNumber) map[string]interface{} {
+	blockNumber := uint64(0)
+	if num == nil || *num == rpc.LatestBlockNumber || *num == rpc.PendingBlockNumber {
```

Comment:
Please don't forget describing in KlaytnDocs that the API supports only "lastest" value as a block parameter.
Maybe we need to refine the use of "earliest", "pending" values in the block parameter someday.

Reply:
@aidan-kwon Understood. And API documentation also have to be updated. I will update it considering your advice. Thank you!